esp-hal/MIGRATING-0.20.md

```diff
@@ -1,8 +1,19 @@
-Migration Guide from 0.20.x to vNext
-====================================
+# Migration Guide from 0.20.x to vNext
```

Comment:
Aww

Reply:
Editor has format on save enabled, all the other markdown files are like this. Can revert if needed but I would rather not.

Reply:
Nah I'll not fight The Formatter, I'm just sad :)